### ISSUE TITLE
Phenotyping process page: needs Late Adult protocol

### ISSUE THREAD

**Issue by viomunoz** [mpi2/PhenotypeData#366]
Requirements: include LA phenotyping procotol information
https://dev.mousephenotype.org/understand/the-data/phenotyping-process-impress/

**Comment by amccoy95** [comment]
Piia will be creating this content, we're not expecting massive paragraphs. 

She's able to to look at this next week

**Comment by viomunoz** [comment]
I agree, may be one short paragraph is enough. For inspiration and/or to include as reference, there is some information about this in the Late Adult heatmap page https://www.mousephenotype.org/understand/data-focus/late-adult-data/ and in the Documentation: https://www.mousephenotype.org/help/standardized-mouse-phenotyping/pipelines/late-adult-and-interval-pipelines/
May be this could potentially be a good time to look at the update of the figure in IMPReSS, that we discussed ealier, here: https://www.mousephenotype.org/impress
